public/index.html

```diff
@@ -26,11 +26,14 @@
     -->
 
     <title>React App</title>
-
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
+    <script
```

Comment:
Do not use CDN instead use packages

https://fontawesome.com/v5/docs/web/use-with/react